src/components/Drawer/readme.md

```diff
@@ -0,0 +1,327 @@
+    Button,
+    Avatar,
+    Drawer,
+    DatePicker,
+    Input,
+    GoogleAddressLookup,
+    Select,
+} from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const StyledExampleHeader = styled.h3.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    font-family: 'Lato';
+    font-size: 36px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.19px;
+    text-align: center;
+    color: ${props => props.text.main};
+`;
+
+const StyledHeader = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    text-align: center;
+    color: ${props => props.text.main};
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 100px;
+    height: 100px;
+    box-shadow: 0 2px 4px 0 rgba(6, 28, 63, 0.3);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+`;
+
+const NameLabel = styled.label.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 16px;
+    font-weight: 900;
+    text-align: center;
+    color: ${props => props.palette.text.main};
+    display: block;
+`;
+
+const EmailLabel = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    font-weight: 300;
+    text-align: center;
+    color: ${props => props.palette.text.label};
+    display: block;
+`;
+
+const DetailsButton = styled(Button)
+`
+    font-size: 0.85rem;
+`;
+
+const ConctactContainer = styled.div`
+    width: max-content;
+`;
+
+const FormNameLabel = styled.label.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 24px;
+    font-weight: 900;
+    line-height: normal;
+    color: ${props => props.palette.text.main};
+    display: block;
+`;
+
+const FormEmailLabel = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 16px;
+    font-weight: 300;
+    line-height: 1.06;
+    color: ${props => props.palette.text.header};
+    display: block;
+`;
+
+const StyledFooterButton = styled(Button)
+`
+    width: 100px;
+`;
+
+const closeDrawer = () => setState({ isOpen : false });
+
+function Contact({ avatar, name, email, onShowDetails }) {
+    return (
+        <ConctactContainer className="rainbow-m-around_large">
+            <div className="rainbow-p-around_medium">
+                <StyledAvatar
+                    src={avatar}
+                    assistiveText={name}
+                    title={name}
+                />
+            </div>
+            <NameLabel className="rainbow-m-top_small">
+                {name}
+            </NameLabel>
+            <EmailLabel className="rainbow-m-top_x-small">
+                {email}
+            </EmailLabel>
+            <div className="rainbow-flex rainbow-justify_space-around">
+                <DetailsButton
+                    variant="base"
+                    label="Details"
+                    onClick={onShowDetails}
+                />
```

Comment:
in this example accessibility is not working as expected because the focus is not being returned to trigger element